### DISCUSSION TITLE
To add Label option

### DISCUSSION THREAD

**Discussion by Tej-Singh-Rana** [community/community#3095] [Discussions]
Hello,  Team
Can you please provide an option to add **short label** in the discussion title or vice versa in the title box?

**Comment by evi-liu** [comment]
Can you please expand on what the short label would entail and what context it would give?

**Comment by ShixiangWang** [comment]
Could any label can be supported, just like issues?

**Comment by PranamBhat** [comment]
Hi @Tej-Singh-Rana 

It is not available at this moment.

However, you can choose a category or click on relevant filter's to get the right discussions. These filters act more as a "label" under GitHub Discussion.

Thank you!
